### PR TITLE
[ACS-4369] Support for PRIMARYPARENT and PARENT fields.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
These fields will go live in 23.1 and so the changes in this PR should wait until it is released.

### CODE REVIEW

**Review by mjstanford — DISMISSED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

File: search-enterprise/latest/using/unsupported.md [hunk 1]
```diff
@@ -68,11 +66,11 @@ Search Enterprise focuses on the most commonly used features, and in some cases
...
-| -------- | ----------------- |
+| --------- | ----------------- |
 | QNAME:'comment' | TYPE:'fm:post' |
 | PNAME:'0/wiki' | PATH:'//cm:wiki/*' |
-| NPATH:'2/Company Home/Sites/swsdp' | PATH: '/app:company_home/st:sites/cm:swsdp//*' |
-| PATH:'//cm:CategoryA/member' | PATH: '//cm:CategoryA/*' AND !TYPE:'cm:category' |
+| NPATH:'2/Company Home/Sites/swsdp' | PATH: '/app:company_home/st:sites/cm:swsdp/*' |
+| ANAME:'0/cdefb3a9-8f55-4771-a9e3-06fa370250f6' | PARENT:'cdefb3a9-8f55-4771-a9e3-06fa370250f6' |
 
 > **Note:** Secondary paths and secondary parents are not supported at this time, so there may still be some differences if these are in use.
```
Comment (mpichura) [inline search-enterprise/latest/using/unsupported.md:75]: That note is not true anymore. With `ESC 4.0.0` we are introducing secondary paths and parents.

File: search-enterprise/latest/using/unsupported.md [hunk 2]
```diff
@@ -57,8 +55,8 @@ The following features, which were supported with Search and Insight Engine 2.x
 
 ### Path Indexing
 
+* Secondary parents
```
Comment (mpichura) [inline search-enterprise/latest/using/unsupported.md:58]: That is not true anymore. With ESC 4.0.0 we are introducing secondary paths and parents.

File: search-enterprise/latest/using/unsupported.md [hunk 3]
```diff
@@ -57,8 +55,8 @@ The following features, which were supported with Search and Insight Engine 2.x
...
 * Secondary paths (paths including secondary parents)
```
Comment (mpichura) [inline search-enterprise/latest/using/unsupported.md:59]: That is not true anymore. With ESC 4.0.0 we are introducing secondary paths and parents.

**Review by mpichura — COMMENTED** [review]
I added some remarks regarding functionalities we managed to introduce with upcoming `ECS 4.0.0` version.
Also we introduced some support for search highlighting, thus line 14 of this file should probably also contain/explain that.

File: search-enterprise/latest/using/unsupported.md (hunk 3 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/using/unsupported.md]: Removed note

File: search-enterprise/latest/using/unsupported.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/using/unsupported.md]: Removed note

File: search-enterprise/latest/using/unsupported.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/using/unsupported.md]: Removed note

**Comment by oliveralfresco2018** [comment]
> I added some remarks regarding functionalities we managed to introduce with upcoming `ECS 4.0.0` version. Also we introduced some support for search highlighting, thus line 14 of this file should probably also contain/explain that.

Thanks for those comments @mpichura 
What do you suggest I add here?
I can remove the unsupported reference but it sounds like you are saying that it is not fully supported? 
Can you provide me with some guidance here? Cheers.

**Review by mjstanford — APPROVED** [review]

**Pull request merged**